src/main/java/com/habicus/core/controller/v1/goal/UserController.java

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Can we add Java Doc to all the public functions in this class?
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html